toolbackup/src/main/java/io/dockstore/tooltester/client/cli/FormattedTimeGenerator.java

```diff
@@ -0,0 +1,31 @@
+package io.dockstore.tooltester.client.cli;
```

Comment:
Oh, also change the package name from `io.dockstore.tooltester` to `io.dockstore.toolbackup`.